### DISCUSSION TITLE
Pypi dependencies and CLI

### DISCUSSION THREAD

**Discussion by boisgera** [prefix-dev/pixi#522] [Q&A] [answered]
Hi everyone !

Is there a way to add pypi dependencies using the command-line ? 

So far I have been editing the `pixi.toml` file directly, but I'd rather use the CLI if it is an option.

Best regards,

Sébastien

P.S. I have barely started using pixi but I really like its workflow so far. Thanks for the hard work! ❤️

**Comment by ruben-arts** [comment, marked as answer]
He @boisgera,

Glad to hear you like it. We are working on multiple stories to improve the `pypi` integration, of which one is the `pixi add` command: https://github.com/prefix-dev/pixi/issues/498

For more topic we are working on follow the [Project](https://github.com/orgs/prefix-dev/projects/6) we made.

**Comment by YeungOnion** [comment]
I noticed that the linked Project didn't mention `pixi global` for pypi-only cli tools. Is there a discussion about this? I occasionally rely on `pipx install` and even less occasionally `pipx inject` and I'm wondering if pixi will have a solution for things that are not conda packages.

[Link to pipx's how-it-works](https://pipx.pypa.io/stable/how-pipx-works/)

**Reply by ruben-arts** [reply]
We indeed didn't work on pypi for `pixi global` there is currently no story to start on that. We probably want to fix this in the future but currently our biggest focus is on project management. Because there we can have the biggest impact :muscle:

**Reply by ruben-arts** [reply]
Btw, this shouldn't be particularly hard so any contributions are welcome!
